plugins/inputs/jti_openconfig_telemetry/openconfig_telemetry.go

```diff
@@ -0,0 +1,324 @@
+package jti_openconfig_telemetry
+
+import (
+	"io"
+	"log"
+	"regexp"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"github.com/influxdata/telegraf/plugins/inputs/jti_openconfig_telemetry/auth"
+	"github.com/influxdata/telegraf/plugins/inputs/jti_openconfig_telemetry/oc"
+	"github.com/influxdata/telegraf/plugins/parsers"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+type OpenConfigTelemetry struct {
+	Server          string
+	Username        string
+	Password        string
+	ClientId        string
+	Sensors         []string
+	SampleFrequency uint32
+	CertFile        string
+	Debug           bool
+	StrAsTags       bool
+
+	parser parsers.Parser
+	sync.Mutex
+
+	// keep the accumulator internally:
+	acc            telegraf.Accumulator
```

Comment:
I don't think you need to save this to the struct, since you pass it to the goroutines from the Start functions.

Reply:
You're right. Don't need it. Changed it accordingly